### ISSUE TITLE
Switch chat messages to status

### ISSUE THREAD

**Issue by TheCodedOne** [ZeroNoRyouki/BigReactors#106]
instead of spamming chat messages like 
![image](https://cloud.githubusercontent.com/assets/17037059/26430476/5386865e-40e5-11e7-806c-e021c085116f.png)

change to status messages like seen here
![image](https://cloud.githubusercontent.com/assets/17037059/26430480/5fe2c1a6-40e5-11e7-951c-c57b6f2d784b.png)

**Comment by ZeroNoRyouki** [comment]
RenderGameOverlayEvent ?

**Comment by TheCodedOne** [comment]
no, its in EntityPlayer, you just do player.sendStatusMessage, the 2nd boolean param is if you want it to show on screen, heres the code for what i showed
![image](https://cloud.githubusercontent.com/assets/17037059/26463963/f9bca8da-417d-11e7-9473-836467c4c12a.png)

**Comment by ZeroNoRyouki** [comment]
uhm that's 1.11+ only by the look of it. However, it's using existing infrastructure so maybe I can simulate it on 1.10 too. 

I'll look into it

**Comment by TheCodedOne** [comment]
in 1.10 i believe you can use `Minecraft.getMinecraft().ingameGUI.setOverlayMessage`

**Comment by TheCodedOne** [comment]
obviously clientside only but some simple packets could be used if you had to run it server side

**Comment by ZeroNoRyouki** [comment]
Server-side is easy because the chat packet already support it (the whole thing it's basically built on vanilla displaying the title of the song being played by the juke box)

**Comment by tyler489** [comment]
Or make text msges stack like bloodmagic does with there alters.
